nym-vpn-core/crates/nym-vpn-lib/src/tunnel_state_machine/route_handler.rs

```diff
@@ -105,6 +105,7 @@ impl RouteHandler {
         let mut routes = HashSet::new();
 
         match routing_config {
+            #[allow(unused)]
```

Comment:
Maybe if you don't use some of the metadata here, then we can either:

- Mark individual properties as unused, i.e `entry_gateway_address: _`
- Or make those fields platform-dependent.